src/jttt/Core/Game.java

```diff
@@ -17,16 +17,16 @@
     private int gameType;
     private HashMap<Mark, Player> players;
 
-    public Game(PlayerFactory playerFactory) {
+    public Game(Board board, PlayerFactory playerFactory) {
         this.playerFactory = playerFactory;
-        this.board = null;
+        this.board = board;
         this.players = new HashMap<>();
         this.gameType = 0;
```

Comment:
0 is a magic number here. The `gameType` variable still doesn't tell me enough about why this is set to `0`.